api/nomics.py

```diff
@@ -0,0 +1,125 @@
+import datetime
+import calendar
+import connexion
+
+from services.bitshares_websocket_client import client as bitshares_ws_client
+from services.bitshares_elasticsearch_client import client as bitshares_es_client
+from services.cache import cache
+import api.explorer
+import es_wrapper
+import config
+
+
+def info():
+    return {
+        "name": "Bitshares",
+        "description": "An exchange description of at least 1000 characters in plain text (no html)",
```

Comment:
You can start with that: https://github.com/bitshares/marketing/tree/master/faq

It should be reviewed to talk aboutht eDEX more than the Blockchain itself.

Reply:
added 1st paragraph by now in https://github.com/bitshares/bitshares-explorer-api/pull/64/commits/5edc75a9e3d55d6c0e86dc2029d2e41cc2bae48b